src/algebra/order/to_interval_mod.lean

```diff
@@ -37,6 +37,41 @@ Unify `smodeq` and `add_comm_group.modeq`, which were originally developed indep
 
 noncomputable theory
 
+namespace add_comm_group
+variables {α : Type*} [add_comm_group α]
+
+/-- `a ≡ b [PMOD p]` means that `b` is congruent to `a` modulo `p`.
+
+Equivalently (as shown further down the file), `b` does not lie in the open interval `(a, a + p)`
+modulo `p`, or `to_Ico_mod hp a` disagrees with `to_Ioc_mod hp a` at `b`, or `to_Ico_div hp a`
+disagrees with `to_Ioc_div hp a` at `b`. -/
+def modeq (p a b : α) : Prop := ∃ z : ℤ, b = a + z • p
+
+notation a ` ≡ `:50 b ` [PMOD `:50 p `]`:0 := modeq p a b
+
+variables {p a b : α}
+
+lemma modeq.symm : a ≡ b [PMOD p] → b ≡ a [PMOD p]
+| ⟨z, hz⟩ := ⟨-z, hz.symm ▸ by rw [neg_smul, add_neg_cancel_right]⟩
+
+lemma modeq_comm : a ≡ b [PMOD p] ↔ b ≡ a [PMOD p] := ⟨modeq.symm, modeq.symm⟩
+
+lemma modeq_iff_eq_add_zsmul : a ≡ b [PMOD p] ↔ ∃ z : ℤ, b = a + z • p := iff.rfl
+
+lemma not_modeq_iff_ne_add_zsmul : ¬a ≡ b [PMOD p] ↔ ∀ z : ℤ, b ≠ a + z • p :=
```

Comment:
I'm surprised that no one seem to have discussed introducing a `≢` notation yet ... (I searched GitHub and Zulip)

Reply:
Are you sure you searched github? #14682

Reply:
Thanks! You can see that no results turn up when you search for the character: https://github.com/leanprover-community/mathlib/search?q=%E2%89%A2 Poor unicode support I guess ...